lib/formulator/input.ex

```diff
@@ -43,10 +43,12 @@ defmodule Formulator.Input do
   defp add_format_validation_attribute(options, _, _), do: options
 
   defp option_enabled?(options, field, default) do
```

Comment:
Given the logic here, do you think it's worth exposing this publicly and unit testing it?

Reply:
I think having the tests on the behavior of the regex options is enough. That said, I think a couple more tests would be good. I'll add some more.